src/workforce-management/Views/Employee/Index.cshtml

```diff
@@ -0,0 +1,22 @@
+﻿@model workforce_management.ViewModels.EmployeeList
+@using Bangazon.Models;
+
+<h2>Employees</h2>
+
+<table class="table table-striped">
+    <thead>
+        <th>Name</th>
+        <th>Department</th>
+        <th>Details</th>
+    </thead>
+    <tbody>
+        @foreach (Employee employee in @Model.Employees)
+            { 
+                <tr>
+                    <td >@Html.DisplayFor(model => employee.FirstName)</td>
+                    <td>@Html.DisplayFor(model => employee.DepartmentId)</td>

```

Comment:
I would change line 17 to this:
`<td>@Html.DisplayFor(model => employee.Department.Name)</td>`